collateral-onboarding/collateral-onboarding/onboard.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+cd ~/spells/spells-mainnet
+dapp update
+make
+dapp create DssSpell
+
+cd ~/spells/exchange-callees
+dapp update
+make
+
+# Deploy mainnet calless
+UniswapV2CalleeDai=$(dapp create UniswapV2CalleeDai \
+  0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D \
+  0x9759A6Ac90977b93B58547b4A71c78317f391A28)
```

Comment:
Where is this code coming from? Why is it copied over to our repo and not cloned/executed directly? Can't you run `https://github.com/makerdao/exchange-callees/blob/3c4055571c0ab17efbd9e7ed809862bc47d8c2fd/Makefile#L7` this instead?